scripts/components/timer.js

```diff
@@ -62,8 +86,8 @@ var TimerPanel = module.exports = React.createClass({
 
         if (selecting) {
             return <div className="gotr-timer-area-container">
-                <div className="gotr-timer-area gotr-timer-area-selecting">
-                    <Selector mins={mins} onChange={this.updateMins}/>
+                <div onKeyDown={this.testFunction} className="gotr-timer-area gotr-timer-area-selecting">
```

Comment:
It might be better to add the `onKeyDown` handler to the input box, so that the escape key will only work then the input box is focused.